contributors.md

```diff
@@ -220,6 +220,12 @@ The following people are not part of the development team, but have been contrib
 * Joel H. (HtotheTML)
 * John Mulcahy (jayjay300)
 * Chase Percy (Chase-Percy)
+<<<<<<< maurer
+=======
+* Wenzhao Qiu (qwzhaox)
+* Michael Bernardi (mrmbernardi)
+* Aram Kazorian (aramk-hub)
+>>>>>>> develop
```

Comment:
This looks like an unresolved merge conflict.

Reply:
I think i fixed these conflict now